apps/ledger-live-mobile/src/reducers/settings.ts

```diff
@@ -287,12 +289,14 @@ const handlers: ReducerMap<SettingsState, SettingsPayload> = {
     },
   }),
 
-  [SettingsActionTypes.SETTINGS_COMPLETE_ONBOARDING]: (state, action) => ({
-    ...state,
-    hasCompletedOnboarding: !!(
-      action as Action<SettingsCompleteOnboardingPayload>
-    ).payload,
-  }),
+  [SettingsActionTypes.SETTINGS_COMPLETE_ONBOARDING]: (state, action) => {
+    return {
+      ...state,
+      hasCompletedOnboarding: !!(
+        action as Action<SettingsCompleteOnboardingPayload>
+      ).payload,
+    };
```

Comment:
Here I think the point of the new payload type `void` was:
- if `completeOnboarding()` is called -> set to true
- leave the option to call completeOnboarding({hasCompletedOnboarding: false})

So here, if payload is undefined, set to true, otherwise use `payload.hasCompletedOnboarding`.
I know it's not your changes but let's fix it properly and check that all dispatches of the action will result in the expected outcome.